src/panels/lovelace/editor/hui-dialog-edit-card.ts

```diff
@@ -1,6 +1,8 @@
 import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
 import { fireEvent } from "../../../common/dom/fire_event.js";
+import yaml from "js-yaml";
 
+import "@polymer/paper-checkbox/paper-checkbox.js";
```

Comment:
not used

Reply:
Without this the checkboxes do not render?

Reply:
there are no checkboxes in this element…